plugins/woocommerce-admin/client/wp-admin-scripts/product-import-tracking/product-import-tracking.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * External dependencies
+ */
+import { getQuery } from '@woocommerce/navigation';
+
+/**
+ * Internal dependencies
+ */
+import {
+	addExitPage,
+	removeExitPage,
+} from '~/customer-effort-score-tracks/customer-effort-score-exit-page';
+
+const ACTION_NAME = 'import_products';
+
+( () => {
+	const query: { step?: string; page?: string } = getQuery();
+
+	if ( query.page !== 'product_importer' ) {
+		return;
+	}
+
+	if ( query.step === 'done' ) {
+		removeExitPage( ACTION_NAME );
+		return;
+	}
+
+	addExitPage( ACTION_NAME );
```

Comment:
So I originally went this route due to the fact that it involved multiple steps with different UI, although you're right that the UI (buttons) are similar enough to where the logic wouldn't be complex at all. I will say that this approach feels a touch more robust since it only interacts with the query string, and not the individual UI elements on each page. 

That said, of course the UI here likely won't change so I'm receptive to changing this if you prefer. There is a benefit to just using the same approach and functions. If we came across another use case that involved multiple steps with differing UI, this capability would be handy, but that could be unlikely. Let me know what you think.

Reply:
Gotcha, I think that it is fine then.
I am trying to think there could be edge cases to this approach, but the only one I can think of is when the actual import fails and it's not directed to the `done` page, but then I guess it would show up and we probably did want the users feedback anyhow.
So I am fine with this, I mostly was pushing back as I don't want to many different ways of handling this, but your way isn't to far out of the way :)